conf/command.conf

```diff
@@ -0,0 +1,95 @@
+## Custom Command
+## CustomCommand {
+##    command {                                      #command string, must be lowercase
+##        [Mode read|write|admin[|[keyAt2|keyAt3]]   #default write, default key position is 1
+##        [MinArgs [2-]]                             #default 2, including command itself
+##        [MaxArgs [2-]]                             #default 2, must be MaxArgs >= MinArgs
+##    }...
+## }
+
+## This section must be defined before Latency monitor to support all commands
```

Comment:
We can delay **setLatencyMonitor** in Conf.cpp to avoid this restriction.

Reply:
Yes, applied.